oas_docs/output/kibana.serverless.yaml

```
@@ -113,7 +113,12 @@ tags:
   - description: Manage and interact with Security Assistant resources.
     name: Security AI Assistant API
     x-displayName: Security AI assistant
-  - description: You can create rules that automatically turn events and external alerts sent to Elastic Security into detection alerts. These alerts are displayed on the Detections page.
+  - description: |-
+      You can create rules that automatically turn events and external alerts sent to Elastic Security into detection alerts. These alerts are displayed on the Detections page.
```

This is the description for security detections APIs right? Could we do something like:

```suggestion
      Use the detection APIs to create and manage rules that search events and external alerts sent to Elastic Security and generate detection alerts from any hits. Alerts are displayed on the Security > Alerts page and can be assigned and triaged, using the alert status to mark them as open, closed, or acknowledged.
```